### PR TITLE
Add renderMesh bias for v3.5.0

### PR DESCRIPTION
**Pull request by troublemaker52025**
Re: cocos-creator/3d-tasks#

Changelog:
 *

### CODE REVIEW

**Comment by troublemaker52025** [comment]
* https://github.com/cocos-creator/3d-tasks/issues/8538

**Review by jk20012001 — APPROVED** [review]

File: cocos/3d/lights/directional-light-component.ts [hunk 1]
```diff
@@ -296,16 +296,16 @@ export class DirectionalLight extends Light {
...
-    @property({ group: { name: 'ShadowSettings', displayOrder: 13 } })
+    @property({ group: { name: 'DynamicShadowSettings', displayOrder: 13 } })
     @editable
     @type(CCFloat)
-    get fixedFar () {
+    get shadowFar () {
         return this._shadowFar;
     }
-    set fixedFar (val) {
+    set shadowFar (val) {
```
Comment (pandamicro) [inline cocos/3d/lights/directional-light-component.ts:305]: Renaming need backward compatibility, including 

1. Declare `@formerlySerializedAs` to ensure data migration
2. Add replaceProperty in module's deprecated

File: cocos/core/renderer/scene/directional-light.ts [hunk 2]
```diff
@@ -207,10 +207,10 @@ export class DirectionalLight extends Light {
       * @en get or set fixed area shadow
       * @zh 是否是固定区域阴影
       */
-    get fixedArea () {
+    get shadowFixedArea () {
         return this._shadowFixedArea;
     }
-    set fixedArea (val) {
```
Comment (pandamicro) [inline cocos/core/renderer/scene/directional-light.ts:213]: Need deprecation

**Review by pandamicro — CHANGES_REQUESTED** [review]
More verification to do

- [ ] Search in editor code the usage of renamed APIs
- [ ] Check MaterialUpgrade scene in test case to check whether old effect can upgrade
- [ ] Ensure backward compatibility for API usage and data migration

File: cocos/core/renderer/scene/directional-light.ts (hunk 2 above)
Comment (troublemaker52025) [reply cocos/core/renderer/scene/directional-light.ts:213]: 这是 “v3.5.0 shadow 属性迁移至灯光组件” 时的命名，v3.5.0 还未发布，也未有用户调用到 fixedArea 接口，故更名不需要做迁移，也不需要废弃接口申明

**Comment by troublemaker52025** [comment]
>

**Comment by troublemaker52025** [comment]
> More verification to do
> 
>     * [ ]  Search in editor code the usage of renamed APIs
> 
>     * [ ]  Check MaterialUpgrade scene in test case to check whether old effect can upgrade
> 
>     * [ ]  Ensure backward compatibility for API usage and data migration

 * [x]  Search in editor code the usage of renamed APIs * https://github.com/cocos-creator/editor-3d/pull/6632
 * [x] Check MaterialUpgrade scene in test case to check whether old effect can upgrade
 * [x]  Ensure backward compatibility for API usage and data migration [API usage] * https://github.com/cocos-creator/test-cases-3d/pull/580, 新增属性，不需要数据迁移，默认值为 0

**Review by pandamicro — APPROVED** [review]

**Pull request merged**